spec/forms/search_developer_form_spec.rb

```diff
@@ -3,16 +3,92 @@
 
 RSpec.describe SearchDeveloperForm do
   describe '#search' do
+    before do
+      @developer1 = create(:developer, email: 'abc2321@gmail.com')
+      @developer2 = create(:developer, email: 'abc2121@gmail.com')
+      @developer3 = create(:developer, email: 'abc2341@gmail.com')
+    end
     context 'with no params' do
+      it 'returns no developers' do
+        form = SearchDeveloperForm.new({})
+        expect(form.search.size).to eq(0)
+      end
+    end
```

Comment:
Please insert a blank line between `end` and `context`.
Otherwise, the code looks dense.

Reply:
I fixed that problem but github don't recognise, could you check here: https://github.com/thupv/FindDeveloper/pull/1/files